users/discord/components/LoadingSpinner.js

```diff
@@ -0,0 +1,7 @@
+const { createElement } = react;
```

Comment:
This is a vanilla JS project, so I think it’s better not to use React. Instead, we can use createElement and other vanilla JS DOM methods.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement